content/swagger/resources/measurement_families/routes/measurement_families.yaml

```diff
@@ -163,3 +163,46 @@ get:
       $ref: "#/responses/403Error"
     406:
       $ref: "#/responses/406Error"
+patch:
+  summary: Update/create several measurement families
+  operationId: "patch_measurement_families"
+  tags:
+    - Measurement family
+  x-versions:
+    - "Serenity"
+  description: This endpoint allows you to update and/or create several measurement families at once.
+  x-body-by-line: Contains a JSON array. Each element of the JSON array is a measurement family in the standard format
```

Comment:
Nope, not how it should be documented. This property can only be used when the body is a stream. Which is not the case here. Look at the reference entity record for an example.

Reply:
```suggestion

```